### DISCUSSION TITLE
How do I delete index has been created, in the  folder, there is no option to delete after selected album

### DISCUSSION THREAD

**Discussion by xxxxzui** [photoprism/photoprism#3831] [Q&A]
How do I delete index has been created, in the document holder, there is no option to delete after selected album
![屏幕截图 2023-10-17 203917](https://github.com/photoprism/photoprism/assets/89773597/a7a90aa2-7bf6-4bb6-b9ce-196714eb5a68)

**Comment by graciousgrey** [comment]
To permanently delete files from PhotoPrism and your file system, you must first move them to the archive. It is not yet possible to move an entire folder to the archive. However, you can open the folder, select all the files, and move them to the archive. In the archive you can then delete the files.

Alternatively, you can delete the files directly in your file system, PhotoPrism will then recognize that the files do not exist anymore during the next indexing.

See also:
https://docs.photoprism.app/user-guide/organize/archive/
https://docs.photoprism.app/user-guide/organize/delete/
https://docs.photoprism.app/user-guide/navigate/#selection-mode-and-multi-select
